volatility3/framework/plugins/linux/pagecache.py

```diff
@@ -420,33 +420,29 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
         ]
 
     @staticmethod
-    def write_inode_content_to_file(
-        inode: interfaces.objects.ObjectInterface,
-        filename: str,
-        open_method: Type[interfaces.plugins.FileHandlerInterface],
-        vmlinux_layer: interfaces.layers.TranslationLayerInterface,
-    ) -> None:
-        """Extracts the inode's contents from the page cache and saves them to a file
+    def read_inode_content(
+            inode: interfaces.objects.ObjectInterface,
+            vmlinux_layer: interfaces.layers.TranslationLayerInterface
```

Comment:
In general, I much prefer that methods take a context and a simple string rather than a full module (which carries around it own copy of the context) to ensure that it's very clear which context is in use (and plugins don't suddenly go weird because an object was made in a different place).  At the moment, we don't do that for `write_inode_content_to_file` and I'm not sure why I let that slide, but given this is a new function, I'd much prefer to take a context and a layer name and then do the lookup, to make everything more explicit as to what's going on please.